.ci/jobs.t/elastic+elasticsearch+periodic+dra-snapshots.yml

```diff
@@ -0,0 +1,45 @@
+---
+- job:
+    name: elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    workspace: /dev/shm/elastic+elasticsearch+%BRANCH%+periodic+dra-snapshot
+    display-name: "elastic / elasticsearch # %BRANCH% - DRA snapshot"
+    description: "Publishing Daily Releasable Artifacts (DRAs) of Elasticsearch %BRANCH% snapshots.\n"
+    builders:
+      - inject:
+          properties-file: '.ci/java-versions.properties'
+          properties-content: |
+            JAVA_HOME=$HOME/.java/$ES_BUILD_JAVA
+            RUNTIME_JAVA_HOME=$HOME/.java/$ES_RUNTIME_JAVA
+      - shell: |
+            #!/usr/local/bin/runbld --redirect-stderr
+            DRA_VERSION=`cat build-tools-internal/version.properties \
+                  | grep elasticsearch \
+                  | sed "s/elasticsearch//g" \
+                  | sed "s/ //g" \
+                  | sed "s/=//"`
+            beatsBuildId=`./.ci/scripts/resolve-dra-manifest.sh beats master`
```

Comment:
Should we use the convention of upper-snake-case for these instead of camel-case?

Reply:
fixed